CMakeLists.txt

```diff
@@ -1446,6 +1447,7 @@ if(WITH_TESTS)
         table/table_test.cc
         table/block_fetcher_test.cc
         test_util/testutil_test.cc
+        util/compression_test.cc
```

Comment:
Can you add this to between util/coding_test.cc and util/crc32c_test.cc to obey alphabetical order?